macros/edr/dbt_artifacts/upload_dbt_exposures.sql

```diff
@@ -1,8 +1,15 @@
 {%- macro upload_dbt_exposures() -%}
+    {% set identifier = 'dbt_exposures' %}
+    {% if results and elementary.get_result_node('model.elementary.%s' % identifier) %}
+      {{ elementary.debug_log('[%s] Artifacts already ran.' % identifier) }}
+      {{ return(none) }}
+    {% endif %}
+
+    {% set relation = elementary.get_elementary_relation(identifier) %}
     {% set edr_cli_run = elementary.get_config_var('edr_cli_run') %}
     {% if execute and not edr_cli_run %}
         {% set exposures = graph.exposures.values() | selectattr('resource_type', '==', 'exposure') %}
-        {% do elementary.upload_artifacts_to_table(this, exposures, elementary.get_flatten_exposure_callback()) %}
+        {% do elementary.upload_csv_artifacts_to_table(relation, exposures, elementary.get_flatten_exposure_callback()) %}
```

Comment:
Do we want to have a flag that uses the previous artifacts uploading?

Reply:
The thing I'm not quite sure how to solve in terms of the compatibility, is that I've made the artifact models `incremental` so that their contents won't necessarily be deleted when the model is run and by that force the re-upload of the same data. A common case is that the artifacts don't change at all.

So the questions is - Is there a way to make a `table` materialization that doesn't delete its contents? (I'll look into it). Another solution is to just make the previous artifacts uploading delete from the table as well.

Reply:
See comment on the materialization below